schema/DISBURSEMENT_PARENTS.json

```diff
@@ -0,0 +1,538 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/DISBURSEMENT_PARENTS.json",
+    "version": "8.3.0.1",
+    "title": "FEC Payroll and Credit Card Disbursements",
+    "description": "SCHEDULE B - Credit Card Payment for Operating Expenditure, Payment to Payroll for Operating Expenditure, (Line 29b) Credit Card Payment for Other Disbursement, Payment to Payroll for Other Disbursement (Line 29)",
+    "type": "object",
+    "required": [
+        "form_type",
+        "filer_committee_id_number",
+        "transaction_type_identifier",
+        "transaction_id",
+        "entity_type",
+        "payee_organization_name",
+        "payee_street_1",
+        "payee_city",
+        "payee_state",
+        "payee_zip",
+        "expenditure_date",
+        "expenditure_amount",
+        "aggregate_amount",
+        "aggregation_group",
+        "expenditure_purpose_descrip"
+    ],
+    "properties": {
+        "form_type": {
+            "title": "FORM TYPE",
+            "description": "",
+            "enum": [
+                "SB21b",
+                "SB29"
+            ],
+            "examples": [
+                "SB21b"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "FORM TYPE",
+                "TYPE": "A/N-8",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "SB21b",
+                "VALUE_REFERENCE": "SB[line# ref]",
```

Comment:
In spec is [SB21b | SB29]

Reply:
Thanks for the feedback; these issues have been addressed!